package.json

```diff
@@ -15,11 +15,12 @@
     "react-scripts": "^3.4.3",
     "reactjs-popup": "^1.5.0",
     "simple-react-modal": "^0.5.1",
-    "styled-components": "^5.1.0"
+    "styled-components": "^5.1.0",
+    "secure-ls": "^1.2.6"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "REACT_APP_SC_DISABLE_SPEEDY=false react-scripts --max_old_space_size=8192 build",
+    "build": "REACT_APP_SC_DISABLE_SPEEDY='false' react-scripts --max_old_space_size=8192 build",
```

Comment:
IMO that doesn't make a difference wrt #74 
to actually disable speedy mode, it would have to be something like `REACT_APP_SC_DISABLE_SPEEDY=''`